repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -1016,6 +1020,13 @@ class SparkILoop(
     sparkContext
   }
 
+  @DeveloperApi
+  def createSQLContext(): SQLContext = {
+    sqlContext = new SQLContext(sparkContext)
```

Comment:
@pwendell I guess your idea is to check whether hive is built with spark.
We need check HiveContext class exists or not in class loader.
1. If it does.  we provide HiveContext to be `sqlContext` as default context (hive dialect)?
2. If doesn't.  we provide o.a.s.s.SQLContext as default context.(sql dialect)?
3. Or do we need provide both `sqlContext` and `hiveContext` in spark shell, user can choose what they like ? 
Sorry if I'm wrong.